cloud_run.tf

```
@@ -0,0 +1,37 @@
+data "google_service_account" "cloudrun_service_account" {
+  account_id = var.service_account_name
+  depends_on = [ google_project_service.iam ]
+}
+
+resource "google_cloud_run_v2_service" "cloud_run"{
+  name = var.cloud_run_name[count.index]
+  location = var.location
```

```suggestion
  location = var.region
```
Now in the dev we have region variable.